filament/src/components/LightManager.cpp

```diff
@@ -482,6 +482,13 @@ float LightManager::getSpotLightOuterCone(Instance i) const noexcept {
     return upcast(this)->getSpotParams(i).outerClamped;
 }
 
+float LightManager::getSpotLightInnerConeApproximate(Instance i) const noexcept {
```

Comment:
Why is this called "approximate"?

Reply:
Because this function is recomputing the inner spot angle based on the current `SpotParams`. This results in some rounding errors for retrieving the inner cone angle.
The alternative would be to store the inner cone angle inside the `SpotParams` when it's being set.

Reply:
If that data is not passed to the shaders, that's fine.